tests/framework/matcher/conditions.go

```diff
@@ -0,0 +1,195 @@
+package matcher
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"kubevirt.io/kubevirt/tests/framework/matcher/helper"
+)
+
+func ConditionMissingOrFalse(conditionType interface{}) types.GomegaMatcher {
```

Comment:
I was undecided if best working with `Or` in the sense of 

```
Should(Or(
HaveConditionMissing(),
HaveConditionFalse(),
))
```

but I think we reached a critical mass which justifies having this combined check. :+1:  I still wonder: I see some tests which were explicitly checking for false which are now more generous.

Reply:
https://github.com/kubevirt/kubevirt/pull/7922#discussion_r1021926217 I don't remember if there is another place where we would need it. I am ok with splitting them and having the merged one at the same time.